Libplanet/Net/SwarmOptions.cs

```diff
@@ -75,5 +75,11 @@ public class SwarmOptions
         /// using an approximated value.
         /// </summary>
         public int BranchpointThreshold { get; set; } = 10;
+
+        /// <summary>
+        /// The minimum number to select from routing table when broadcast messages.
+        /// It is 10 by default.
+        /// </summary>
+        public int MinimumBroadcastTarget { get; set; } = 10;
```

Comment:
Shouldn't this be recorded in the changelog?

Reply:
I was intending to write a changelog when importing to the main branch.

Reply:
👌🏼